bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableIO.java

```diff
@@ -842,7 +898,14 @@ public PDone apply(PCollection<T> input) {
   public static PTransform<PCollection<Mutation>, PDone> writeToTable(
       CloudBigtableTableConfiguration config) {
     validate(config, config.getTableId());
-    return new CloudBigtableWriteTransform<>(new CloudBigtableSingleTableWriteFn(config));
+
```

Comment:
Nit: Do you think the following is clearer to read?

DoFn<Mutation, Void> writeFn = null;
if (...) {
  writeFn = new CloudBigtableSingleTableSerialWriteFn(config);
} else {
 writeFn = CloudBigtableSingleTableBufferedWriteFn(config);
}
return new CloudBigtableWriteTransform<>(writeFn);